### ISSUE TITLE
Flag -site24x7.token is not being accepted

### ISSUE THREAD

**Issue by mdiazv** [echocat/site24x7_exporter#4]
I just updated to the new v0.1.7 docker image and it fails to run the way v0.1.6 did:

My entrypoint is: 
```
/usr/bin/site24x7_exporter -site24x7.token "the real token"
```

These are the logs:
```
7/15/2019 5:35:29 PMsite24x7_exporter (version: v0.1.7, revision: 91d540820084608aa08b78a7baf3a735f33d6e45, build: 2019-07-15T21:35:29Z)
7/15/2019 5:35:29 PMURL: https://github.com/echocat/site24x7_exporter
7/15/2019 5:35:29 PMAuthor(s): Gregor Noczinski (gregor@noczinski.eu)
7/15/2019 5:35:29 PM
7/15/2019 5:35:29 PMError: flag provided but not defined: -site24x7.token
7/15/2019 5:35:29 PM
7/15/2019 5:35:29 PMUsage: /usr/bin/site24x7_exporter <flags>
7/15/2019 5:35:29 PMFlags:
7/15/2019 5:35:29 PM  -site24x7.timeout duration
7/15/2019 5:35:29 PM    	Timeout for trying to get stats from site24x7. (default 5s)
7/15/2019 5:35:29 PM  -site24x7.token string
7/15/2019 5:35:29 PM    	Token to access the API of site24x7.
7/15/2019 5:35:29 PM    		See: https://www.site24x7.com/app/client#/admin/developer/api
7/15/2019 5:35:29 PM  -web.listen-address string
7/15/2019 5:35:29 PM    	Address to listen on for web interface and telemetry. (default ":9112")
7/15/2019 5:35:29 PM  -web.telemetry-path string
7/15/2019 5:35:29 PM    	Path under which to expose metrics. (default "/metrics")
7/15/2019 5:35:29 PM  -web.tls-cert string
7/15/2019 5:35:29 PM    	Path to PEM file that conains the certificate (and optionally also the private key in PEM format).
7/15/2019 5:35:29 PM    		This should include the whole certificate chain.
7/15/2019 5:35:29 PM    		If provided: The web socket will be a HTTPS socket.
7/15/2019 5:35:29 PM    		If not provided: Only HTTP.
7/15/2019 5:35:29 PM  -web.tls-client-ca string
7/15/2019 5:35:29 PM    	Path to PEM file that conains the CAs that are trused for client connections.
7/15/2019 5:35:29 PM    		If provided: Connecting clients should present a certificate signed by one of this CAs.
7/15/2019 5:35:29 PM    		If not provided: Every client will be accepted.
7/15/2019 5:35:29 PM  -web.tls-private-key string
7/15/2019 5:35:29 PM    	Path to PEM file that contains the private key (if not contained in web.tls-cert file).
```

Update: I tested other flags and they all do fail, with a similar message.

**Comment by blaubaer** [comment]
Sorry for introducing this stupid bug. A fix is available with version `v0.1.8`.

**Comment by mdiazv** [comment]
In production, right now. Thanks!
